### ISSUE TITLE
Treetable with multiline header (fixed height) hides first row(s)

### ISSUE THREAD

**Issue by derkoe** [porscheinformatik/clarity-addons#840]
![treetable-multiline-header-fixed](https://user-images.githubusercontent.com/123199/88390199-32e89000-cdb8-11ea-98ac-6c75440ab79b.JPG)

**Comment by derkoe** [comment]
Workaround:
```css
.treetable-host .treetable .treetable-header {
  min-height: unset; // workaround for multi-line sticky header
}
```

**Comment by Wallabeng** [comment]
Fixed by #842
